pkg/build/api/v1/types.go

```diff
@@ -161,6 +161,8 @@ type BuildSource struct {
 	Git *GitBuildSource `json:"git,omitempty" description:"optional information about git build source"`
 
 	// Image describes an image to be used to provide source for the build
+	// EXPERIMENTAL.  This will be changing to an array of images in the near future
+	// and no migration/compatibility will be provided.  Use at your own risk.
 	Image *ImageSource `json:"image,omitempty" description:"optional image build source"`
```

Comment:
Do you mind adding it to the description tag on this line so that `api/swagger-spec/oapi-v1.json` is updated.